src/apps/sources/components/SourceForm.tsx

```diff
@@ -225,28 +284,29 @@ const SourceForm: React.FC<Props> = ({
                                 <ErrorMessage name="supported_locales" component="span" />
                             </Typography>
                         </FormControl>
-                        <Field
-                            // required
-                            fullWidth
-                            multiline
-                            rowsMax={4}
-                            autoComplete="off"
-                            id="custom_validation_schema"
-                            name="custom_validation_schema"
-                            label="Custom Validation Schema"
-                            margin="normal"
-                            component={TextField}
-                        />
-                        <Field
-                            // required
-                            fullWidth
-                            autoComplete="off"
-                            id="external_id"
-                            name="external_id"
-                            label="External Id"
-                            margin="normal"
-                            component={TextField}
-                        />
+                        {viewing ? (
```

Comment:
can we extract this block in a function above, and call the function here. return will be much cleaner that way

Reply:
@rakesh-1010 Done !